cert/openssl.cnf

```diff
@@ -135,260 +140,366 @@ caIssuers;URI.0 = http://cacerts.digicert.com/WinnForumCA.crt
 OCSP;URI.0 = http://ocsp.digicert.com
 
 ####################################################################
-[ wif_root_ca ]
+[ root_ca ]
 
 subjectKeyIdentifier	= hash
 authorityKeyIdentifier	= keyid:always,issuer
 basicConstraints 		= critical, CA:true
 keyUsage 				= critical, digitalSignature, cRLSign, keyCertSign
+certificatePolicies=@cps,ROLE_CA
+
+[ cps ]
 
-ROLE = ASN1:UTF8String:CA
+policyIdentifier = 2.16.840.1.114412.2.1
+ CPS.1="https://www.digicert.com/CPS"
 
 ####################################################################
-[ wif_sas_ca ]
+[ sas_ca ]
 
 subjectKeyIdentifier = hash
 #authorityKeyIdentifier = keyid:always,issuer
 basicConstraints = critical, CA:true, pathlen:0
 keyUsage = critical, digitalSignature, cRLSign, keyCertSign
+certificatePolicies=@cps,ROLE_CA,ROLE_SAS
 
-ROLE=ASN1:SEQUENCE:sas_seq
+[ cps ]
 
-[ sas_seq ]
-field1 = UTF8:SAS
-field2 = UTF8:CA
+policyIdentifier = 2.16.840.1.114412.2.1
+ CPS.1="https://www.digicert.com/CPS"
 
-[ wif_sas_ca_sign ]
+[ sas_ca_sign ]
 
 subjectKeyIdentifier = hash
 authorityKeyIdentifier = keyid:always,issuer
 basicConstraints = critical, CA:true, pathlen:0
 keyUsage = critical, digitalSignature, cRLSign, keyCertSign
+certificatePolicies=@cps,ROLE_CA,ROLE_SAS
 
-ROLE=ASN1:SEQUENCE:sas_seq
+[ cps ]
 
-[ sas_seq ]
-field1 = UTF8:SAS
-field2 = UTF8:CA
+policyIdentifier = 2.16.840.1.114412.2.1
+ CPS.1="https://www.digicert.com/CPS"
 
 ####################################################################
-[ wif_oper_ca ]
+[ oper_ca ]
 
 subjectKeyIdentifier = hash
 basicConstraints = critical, CA:true, pathlen:0
 keyUsage = critical, digitalSignature, cRLSign, keyCertSign
+certificatePolicies=@cps,ROLE_CA,ROLE_OPERATOR
 
-ROLE=ASN1:SEQUENCE:oper_seq
+[ cps ]
 
-[ oper_seq ]
-field1 = UTF8:OPERATOR
-field2 = UTF8:CA
+policyIdentifier = 2.16.840.1.114412.2.1
```

Comment:
Yeah, just a place holder using DigiCert's general CPS OID. 
If WINNF were to have a global policy OID, this could be replaced by that. Most likely each CA would also want to include their specific, applicable policy as well. 
For a WINNF global policy OID, I think I would propose using something like 1.3.6.1.4.1.46609.2.1. WINNF owns the entire "1.3.6.1.4.1.46609" structure, correct?